src/field_theory/minpoly/field.lean

```diff
@@ -4,8 +4,9 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Riccardo Brasca, Johan Commelin
 -/
 import data.polynomial.field_division
-import ring_theory.integral_closure
 import field_theory.minpoly.basic
+import ring_theory.algebraic
+import ring_theory.integral_closure
```

Comment:
Technically, `ring_theory.algebraic` already imports `integral_closure`. (Should I conclude from the fact that we now need this extra import of `ring_theory.algebraic` that the import of `ring_theory.integral_closure` was actually unnecessary before this PR since it was included in `algebraic` which was included in some transitive import of this file?)
```suggestion
```

Reply:
That sounds sensible!